src/extensions/button-controls/styles/style.scss

```diff
@@ -1,6 +1,6 @@
 body{
 	.wp-block-button {
-		&.btn-is-full{
+		&.w-100{
```

Comment:
Let's pull this out of the `.wp-block-button` so that we can use it elsewhere where we need to apply 100% width as well.

Reply:
Nice call! Let me check if that will on all instances 👍

Reply:
@richtabor Changed :) Let me know your thoughts. Thanks!